mysql-test/main/backup_locks.test

```diff
@@ -188,6 +188,79 @@ DROP TABLE t3;
 BACKUP UNLOCK;
 DROP TABLE t3;
 
+--echo #
+--echo #  MDEV-28367: BACKUP LOCKS on table to be accessible to those
+--echo #              with database LOCK TABLES privileges
+--echo #
+
+--source include/have_metadata_lock_info.inc
+create database db1;
+create table db1.t1(t int);
+create user user1@localhost;
+select user,host from mysql.user where user='user1';
+--error ER_DBACCESS_DENIED_ERROR
+--connect (con1, localhost, user1, ,db1)
+
+# Global privileges should be granted to all schemas, not individual DB
+--error ER_WRONG_USAGE
+grant reload on db1.* to user1@localhost;
```

Comment:
You shouldn't need to set granting of `reload` to a DB.

Reply:
There is no test that is covering this specific scenario and even more specifically, there is no test for `backup locks` that is showing how to use this privilege properly. Maybe should be added in other commit, but I think it is worth to be covered. The same goes to line `210`.

Reply:
There is no test that is covering this specific scenario and even more specifically, there is no test for backup locks that is showing how to use this privilege properly. Maybe should be added in other commit, but I think it is worth to be covered. The same goes to line 210.

Reply:
sure, put them in `main.grant` however.